### PR TITLE
[IMP][8.0]sale_packaging_price: Add material package type

### PR DESCRIPTION
**Pull request by sergio-teruel**

### CODE REVIEW

**Comment by rafaelbn** [comment]
Great, thanks! :+1:

File: sale_packaging_price/i18n/es.po [hunk 1]
```diff
@@ -114,3 +154,12 @@ msgstr ""
...
 msgstr "Arriba"
+
+#~ msgid "Cardboard"
+#~ msgstr "Cartón"
+
+#~ msgid "Metal"
+#~ msgstr "Metal"
+
+#~ msgid "Wood"
+#~ msgstr "Madera"
```
Comment (yajo) [inline sale_packaging_price/i18n/es.po]: Remove these.

File: sale_packaging_price/i18n/sale_packaging_price.pot [hunk 2]
```diff
@@ -6,8 +6,8 @@ msgid ""
 msgstr ""
 "Project-Id-Version: Odoo Server 8.0\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2015-11-13 08:28+0000\n"
-"PO-Revision-Date: 2015-11-13 08:28+0000\n"
+"POT-Creation-Date: 2016-01-28 11:15+0000\n"
+"PO-Revision-Date: 2016-01-28 11:15+0000\n"
```
Comment (yajo) [inline sale_packaging_price/i18n/sale_packaging_price.pot]: Remove this file.

File: sale_packaging_price/models/product.py [hunk 3]
```diff
@@ -14,6 +20,9 @@ class ProductPackaging(models.Model):
         string='Package Price',
         digits_compute=dp.get_precision('Product Price'),
         help="This price will be considered as a price for complete package")
+    package_material = fields.Many2one(
```
Comment (yajo) [inline sale_packaging_price/models/product.py]: `package_material_id`, following the guidelines.

File: sale_packaging_price/models/product.py [hunk 4]
```diff
@@ -14,6 +20,9 @@ class ProductPackaging(models.Model):
...
+        comodel_name='product.packaging.material',
+        string='Package Material Type')
```
Comment (yajo) [inline sale_packaging_price/models/product.py]: `'Package Material'` is enough.

File: sale_packaging_price/views/product_view.xml [hunk 5]
```diff
@@ -55,10 +59,12 @@
         <field name="model">product.packaging</field>
         <field name="arch" type="xml">
             <search string="Search Product Packaging">
-               <field name="product_tmpl_id"/>
-               <group expand="0" string="Group By...">
+                <field name="product_tmpl_id"/>
+                <field name="package_material"/>
```
Comment (yajo) [inline sale_packaging_price/views/product_view.xml]: `package_material_id`

File: sale_packaging_price/views/product_view.xml [hunk 6]
```diff
@@ -55,10 +59,12 @@
...
+                <group expand="0" string="Group By...">
                    <filter string="Product" context="{'group_by': 'product_tmpl_id'}"/>
                    <filter string="Logistic Unit" context="{'group_by': 'ul'}"/>
+                   <filter string="Pack Type" context="{'group_by': 'package_material'}"/>
                </group>
```
Comment (yajo) [inline sale_packaging_price/views/product_view.xml:68]: Please could you try to align all lines at the same indentation stops? Sometimes you use 3 spaces and sometimes 4, and it's a bit annoying and confusing.

File: sale_packaging_price/i18n/es.po [hunk 7]
```diff
@@ -26,16 +26,56 @@ msgid ""
...
 
 #. module: sale_packaging_price
+#: view:product.packaging:sale_packaging_price.product_packaging_filter
+msgid "Pack Type"
+msgstr "Tipo de paquete"
+
+#. module: sale_packaging_price
+#: field:product.packaging.material,name:0
+msgid "Package Material"
+msgstr "Material empaquetado"
```
Comment (yajo) [inline sale_packaging_price/i18n/es.po]: Material **de** empaquetado.

**Comment by sergio-teruel** [comment]
@Yajo Changes done

**Comment by yajo** [comment]
Please add instructions of this changes to README: What, why and how.

**Comment by rafaelbn** [comment]
Please @Yajo last review here, thanks!

**Comment by carlosdauden** [comment]
:+1:

**Comment by yajo** [comment]
:+1:

**Pull request merged**